tracer/src/Datadog.InstrumentedAssemblyVerification.Standalone/Datadog.InstrumentedAssemblyVerification.Standalone.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Company>Datadog Ltd.</Company>
+    <Copyright>Datadog Ltd.</Copyright>
```

Comment:
```suggestion
    <Company>Datadog</Company>
    <Copyright>Copyright 2022 Datadog, Inc.</Copyright>
```

Reply:
Fixed in 4c7c44dcf9bc4f5affbde6c20b8fd17d1735d225